### PR TITLE
Update node util

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

File: services/util/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Service for utils",
   "main": "src/app.js",
   "devDependencies": {
+    "@types/restify-errors": "~4.3.5",
```
Comment (erikasv) [inline services/util/package.json:7]: I think we don't need the types here, and just realize that they aren't needed in ecosystems either

File: services/util/src/services/downloads.js [hunk 2]
```diff
@@ -42,8 +43,7 @@ module.exports = (DownloadsPersistence, CloudServices, logger) => {
       if (cloud[0].storage_service === 'aws') {
         uploadFunction = CloudServices.AWS.uploadFile;
       } else {
-        const Error = { message: 'Service not configured' };
-        throw Error;
+        throw new RestifyErrors.NotFoundError('Service not configured');
```
Comment (erikasv) [inline services/util/src/services/downloads.js:46]: This is an internal error

File: services/util/package.json (hunk 1 above)
Comment (cazapatamar) [reply services/util/package.json]: fixed!

File: services/util/src/services/downloads.js (hunk 2 above)
Comment (cazapatamar) [reply services/util/src/services/downloads.js]: fixed!

**Review by erikasv — APPROVED** [review]

**Pull request merged**